framework/src/util/Properties.h

```diff
@@ -45,6 +45,8 @@ class Properties : public detail::MultiThreaded<>
   std::pair<Any, bool> Value_unlocked(const std::string& key,
                                       bool matchCase = false) const;
 
+  const Any& ValueByRef(const std::string& key, bool matchCase = false) const;
```

Comment:
This should be renamed to `ValueByRef_unlocked` to follow the same convention as `Value_unlocked` to indicate that this function doesn't hold a lock.